challenge_7/python/ajschrier/MissingNumber.py

```diff
@@ -0,0 +1,23 @@
+def builtInMethod(inputList):
+    for i in xrange(len(inputList)):
+        # Check if i is in list
+        if i not in inputList:
+            # Return number
+            return i
+    # Return 0 if none of the numbers meet the condition
+    return 0
```

Comment:
It would be more Pythonic to raise an Exception. Also 0 is in the list of numbers so return -1.